6.4.rememberRemember.py

```diff
@@ -0,0 +1,22 @@
+import os
+import numpy as np
+from PIL import Image
+
+
+# 6.4 זכרו זכרו
+def decode_img(path=None):
+    if not path:
+        path = os.path.join(os.getcwd(), 'resource', 'code.png')
+
+    np_img_trans = np.array(Image.open(path)).transpose()
+
+    ans = []
+    for row in np_img_trans:
```

Comment:
You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs.